woocommerce/woocommerce
# Remove old plugin/theme background installer hooks

### All Submissions:






### Changes proposed in this Pull Request:

This PR removes the woocommerce_plugin_background_installer and woocommerce_theme_background_installer hooks and functionality. Since there is no cron event scheduled for these functions it has no use in the codebase. The Wizard has it's own plugin/theme installer functionality that uses the shutdown hook instead of a cron for installing plugins and themes.

### How to test the changes in this Pull Request:

1. On a clean WC install, use the Wizard and ensure all the plugins and themes you select to be installed, for example Stripe and Storefront, are installed correctly.
2. Check that tests pass.

### Other information:




### Changelog entry

> Remove - Plugin and Theme background installer hooks, cannot run as there are no events scheduled for it.
